examples/text_classification/train.py

```diff
@@ -10,9 +10,30 @@
 
 from model import TextSentiment
 
+r"""
+This file shows the training process of the text sentiment model.
```

Comment:
This works on all text classification datasets not just those relevant to sentiment.